lib/client/bst-config.ts

```diff
@@ -141,6 +141,16 @@ export class BSTProcess {
         return process;
     }
 
+    public kill(): boolean {
+        try {
+            process.kill(this.pid, "SIGKILL");
```

Comment:
SIGKILL is brutal. Instant death. SIGTERM can be caught. You can do this:

process.on('SIGTERM', function () {
  // Gentle cleanup goes here ...  
});